src/OrchardCore.Modules/Orchard.Localization/Startup.cs

```diff
@@ -25,7 +25,7 @@ public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IS
 
         public override void ConfigureServices(IServiceCollection services)
         {
-            services.AddOrchardLocalization();
+            services.AddOrchardLocalization(options => options.ResourcesPath = "Localization");
```

Comment:
Is the default value `"Resources"` if you don't set it to `"Localization"`?
I am not asking to change it as you do it in the module. Just what it would be if someone was using the project directly from MVC.